### PR TITLE
Fix wrong firstConsecutiveStaticIp 

### PR DESCRIPTION
**Pull request by zqingqing1**
**What this PR does / why we need it**: There is a bug in example/e2e-tests/kubernetes/release/default/definition.json. In the MasterProfile, the firstConsecutiveStaticIp is not in the range of vnetCidr. Correct it from "10.239.255.239" to "10.230.255.239".

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: examples/e2e-tests/kubernetes/release/default/definition.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
       "OSDiskSizeGB": 200,
       "vnetSubnetId": "/subscriptions/SUB_ID/resourceGroups/RG_NAME/providers/Microsoft.Network/virtualNetworks/VNET_NAME/subnets/SUBNET_NAME",
       "firstConsecutiveStaticIP": "10.239.255.239",
-      "vnetCidr": "10.230.0.0/16"
+      "vnetCidr": "10.239.0.0/16"
```
Comment (jackfrancis) [inline examples/e2e-tests/kubernetes/release/default/definition.json]: @jchauncey how is the vnet creation handled for the ginkgo e2e runner?

Comment (jchauncey) [reply examples/e2e-tests/kubernetes/release/default/definition.json]: its hard coded right now (see line 107 in `cli_provisioner.go`)

Comment (jchauncey) [reply examples/e2e-tests/kubernetes/release/default/definition.json]:
```
	vnetName := fmt.Sprintf("%sCustomVnet", cli.Config.Name)
	subnetName := fmt.Sprintf("%sCustomSubnet", cli.Config.Name)
	if cli.CreateVNET {
		err = cli.Account.CreateVnet(vnetName, "10.239.0.0/16", subnetName, "10.239.0.0/16")
		if err != nil {
			return fmt.Errorf("Error trying to create vnet:%s", err)
		}
		subnetID = fmt.Sprintf("/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Network/virtualNetworks/%s/subnets/%s", cli.Account.SubscriptionID, cli.Account.ResourceGroup.Name, vnetName, subnetName)
	}
```

**Comment by jackfrancis** [comment]
@jchauncey thanks. @zqingqing1 It looks like the VNET is being created with the `10.239.0.0/16` address space, and a single subnet that comprises the whole `10.239.0.0/16` space. This means we need to change the `clusterSubnet` value as well to be `10.239.0.0/16`. Thanks!

**Comment by zqingqing1** [comment]
I searched for all files, and found that 10.230.0.0/16 occurs at 2 files. One is at this file, another is at examples\kubernetes-config\kubernetes-clustersubnet.json. How clustersubnet in the latter file being used? @jchauncey  @jackfrancis

**Comment by jackfrancis** [comment]
Thanks @zqingqing1 !

**Pull request merged**